### ISSUE TITLE
Fix tabbing in search results

### ISSUE THREAD

**Issue by kacperknapik** [SAP/cloud-commerce-spartacus-storefront#5264]
Possibly `[tabindex]` attribute should be enough, to make that work.

Stretch: replace icon with a button.

NOTE: remember about aux keys (arrows - up and down).

**Comment by giancorderoortiz** [comment]
I see this works.
